cli/src/program.rs

```diff
@@ -1103,6 +1331,162 @@ fn process_dump(
     }
 }
 
+fn close(
+    rpc_client: &RpcClient,
+    config: &CliConfig,
+    account_pubkey: &Pubkey,
+    recipient_pubkey: &Pubkey,
+    authority_signer: &dyn Signer,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let (blockhash, _) = rpc_client.get_recent_blockhash()?;
+
+    let mut tx = Transaction::new_unsigned(Message::new(
+        &[bpf_loader_upgradeable::close(
+            &account_pubkey,
+            &recipient_pubkey,
+            &authority_signer.pubkey(),
+        )],
+        Some(&config.signers[0].pubkey()),
+    ));
+
+    tx.try_sign(&[config.signers[0], authority_signer], blockhash)?;
```

Comment:
will this fail if `config.signers[0]` and `authority_signer` are the same?

Reply:
Doesn't fail if same